README.rst

```diff
@@ -69,6 +69,8 @@ UCI configuration options must go in ``/etc/config/openwisp``.
 - ``test_script``: custom test script, read more about this feature in `Configuration test`_
 - ``uuid``: unique identifier of the router configuration in the controller application
 - ``key``: key required to download the configuration
+- ``hardware_id_script``: custom script to read out a hardware id, read more about this feature in `Hardware ID`_
```

Comment:
I'd add `[...] to read out a hardware id (eg: a serial number) [...]` just to make sure users see it.. you know how we do it.. we read stuff quickly and scan for specific words so a tiny bit of redundancy in docs can be helpful